packages/parmap/parmap.1.1.1/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+  ["dune" "build" "-p" name "@doc"] {with-doc}
+]
+maintainer: ["Roberto Di Cosmo <roberto@dicosmo.org>"]
+authors: ["Roberto Di Cosmo <roberto@dicosmo.org>"]
+bug-reports: "https://github.com/rdicosmo/parmap/issues"
+homepage: "https://github.com/rdicosmo/parmap"
+license: "LGPL-2.1-or-later WITH OCaml-LGPL-linking-exception"
+dev-repo: "git+https://github.com/rdicosmo/parmap.git"
+synopsis: "Minimalistic library allowing to exploit multicore architecture"
+description: """
+Parmap is a minimalistic library allowing to exploit multicore
+architecture for OCaml programs with minimal modifications: if you
+want to use your many cores to accelerate an operation which happens
+to be a map, fold or map/fold (map-reduce), just use Parmap’s parmap,
+parfold and parmapfold primitives in place of the standard List.map
+and friends, and specify the number of subprocesses to use by the
+optional parameter ~ncores."""
+depends: [
+  "dune"
```

Comment:
```suggestion
  "dune" {>= "1.10.0"}
```
According to the `dune-project` file in the root https://github.com/rdicosmo/parmap/blob/1.1.1/dune-project this needs a lower bound